### PR TITLE
Feature about: Update Open graph for facebook and twitter

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
-Add open graph images for twitter and facebook share button

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <meta name="twitter:creator" content="@Code4Africa">
   <meta name="twitter:description" content="Stories from across the continent about FGM. Supported by Code for Africa and ICFJ">
   <meta name="twitter:title" content="#BornPerfect">
-  <meta name="twitter:image" content="https://bornperfect.africa/images/background/Header-banner.jpg">
+  <meta name="twitter:image" content="https://bornperfect.africa/img/background/Header-banner.jpg">
```
Comment (DavidLemayian) [inline _layouts/base.html]: Let's use either relative urls or Liquid template tags e.g `{{ site.url }}` to make it easier to re-use code.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: _layouts/base.html [hunk 2]
```diff
@@ -8,10 +8,10 @@
   <meta name="author" content="{{site.author}}">
 
   <!-- The Open Graph protocol -->
+  <meta property="og:type" content="website">
   <meta property="og:title" content="#BornPerfect">
-  <meta property="og:type" content="article">
-  <meta property="og:image" content="https://bornperfect.africa/images/background/Header-banner.jpg">
   <meta property="og:url" content="https://bornperfect.africa">
+  <meta property="og:image" content="https://bornperfect.africa/img/background/Header-banner.jpg">
```
Comment (DavidLemayian) [inline _layouts/base.html]: Same goes for these.

File: _config.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@ author: Code for Africa.
 
 url: https://bornperfect.africa
 enforce_ssl: bornperfect.africa
+image: /img/background/Header-banner.jpg
```
Comment (DavidLemayian) [inline _config.yml]: @nerdlet Why would we put this here? The label/variable is also not very descriptive.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 3 above)
Comment (nerdlet) [reply _config.yml]: @DavidLemayian I was creating a url image path for the site image.Already foxed it but forgot to remove that line.

**Pull request merged**